### PR TITLE
feat: add rails documentation for migration to v8

### PR DESCRIPTION
**Pull request by nicolasalexandre9**

### CODE REVIEW

File: how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+bundle install
+```
+
+{% hint style="info" %}
+In case of a regression introduced in Production after the upgrade, a rollback to your previous liana version 7 is the fastest way to restore your admin panel.
+{% endhint %}
+
+## Breaking changes
+
+This new version (v8) does not support the old role system. If you are in the case, please follow [this procedure](../migrate-to-the-new-role-system.md) in order to migrate to the new role system **before** you attempt to upgrade to version 8.
```
Comment (SteveBunlon) [inline how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md:37]:
```suggestion
This new version (v8) does not support the old role system. If you are still using the old role system, please follow [this procedure](../migrate-to-the-new-role-system.md) in order to migrate to the new role system **before** you attempt to upgrade to version 8.
```

File: how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+{% hint style="warning" %}
+**How do I know if I'm using the old or new role system?**
+
+If you have access to roles (Project settings > Roles) as designed below...\
+\
+![](<../../../.gitbook/assets/image (10).png>)\
+\
+then you are using the new role system and can go ahead with the upgrade to v9.
```
Comment (SteveBunlon) [inline how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md:46]:
```suggestion
then you are using the new role system and can go ahead with the upgrade to v8.
```

File: how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+{% endhint %}
+
+
+If you were using the ```ForestLiana::PermissionsChecker``` for check the permission on overriding route.
```
Comment (SteveBunlon) [inline how-tos/maintain/upgrade-notes-rails/upgrade-to-v8.md:50]:
```suggestion
In your code, if you were using the ```ForestLiana::PermissionsChecker``` for check the permission on overriding route. 
```

File: reference-guide/routes/override-a-route.md [hunk 4]
```diff
@@ -509,8 +509,7 @@ if ForestLiana::UserSpace.const_defined?('UserController')
 
     # Create a User
     def create
-      checker = ForestLiana::PermissionsChecker.new(@resource, 'addEnabled', @rendering_id, user_id: forest_user['id'])
-      return head :forbidden unless checker.is_authorized?
+	  ForestLiana::Ability::forest_authorize!('add', forest_user, @resource)
```
Comment (SteveBunlon) [inline reference-guide/routes/override-a-route.md:512]: The rendering is automatically fetched from the request now ? Just to confirm 😇

**Review by SteveBunlon — CHANGES_REQUESTED** [review]

File: reference-guide/routes/override-a-route.md (hunk 4 above)
Comment (nicolasalexandre9) [reply reference-guide/routes/override-a-route.md:512]: I can add in the snippet a begin … rescue for return a HTTP response, what do you think ?

**Pull request merged**